paddle/go/pserver/service.go

```diff
@@ -0,0 +1,179 @@
+package pserver
+
+import (
+	"errors"
+	"fmt"
+	"sync"
+)
+
+// ElementType is the type of elements of a Parameter.
+type ElementType int
+
+var ErrUnintialized = errors.New("pserver not initialized")
+var ErrAlreadyIntialized = errors.New("pserver already initialized")
```

Comment:
Spell error: `ErrUnintialized` and `ErrAlreadyIntialized`. Should be "Initialized"